README.md

```diff
@@ -12,14 +12,28 @@ on-chain fee.
 
 More on how this works [here](https://github.com/hsjoberg/blixt-wallet/issues/242).
 
+## Dunder's API
+
+The API endpoints supplied by dunder LSP are:
+
+- /estimateFee - returns estimated fee (sats) and fee rate (sats per byte)
+
+- /getInfo - returns info from lnd - for details, see https://github.com/LN-Zap/node-lnd-grpc/blob/master/proto/0.12.0-beta/lnrpc/rpc.proto#L1567
```

Comment:
Sure - just that by pointing to master for a link that is just my feeble (or naive/minimalistic) attempt at explaining "get info" the line number would need to be updated, so I'll add to the link what to look for.
I found almost no docs at all for get info, so I suspect this is a case of code moving way faster than docs and we'll have to live with this for now. Or maybe better to remove the link and explain in words what /getInfo does and is used for?

Reply:
> I found almost no docs at all for get info

Oh the docs are available here https://api.lightning.community/#getinfo. getInfo is very stable, so we can use this one.

If you want to link to the RPC you can link the file in the source repo instead of Zap: https://github.com/lightningnetwork/lnd/blob/260ea9b842ddd80fbea1df5516f557e3081f743f/lnrpc/rpc.proto#L1596

But I suppose it makes more sense to link to the docs.